Automattic/jetpack
# Add backup video section to connect page

Add video section to backup connect page

## Proposed changes:

* Adds section to backup connect page with same video that is on jetpack.com/backup
* Section also has CTA to get Backup

### Other information:


## Jetpack product discussion

P2: p1HpG7-kd7-p2
Figma: bSsQjyTPK8VIfopwI4fdIU-fi

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch locally or with the Jetpack Beta plugin
2. DONT connect Jetpack to your wordpress account
3. Install the Jetpack VaultPress Backup plugin
4. If you are not redirected there, go to `/wp-admin/admin.php?page=jetpack-backup`
5. See that there is a new section with a video and CTA
![image](https://github.com/Automattic/jetpack/assets/65001528/c8fc216f-a955-46bb-a4e9-0e65212cfed1)
6. Make sure it matches comps and looks good on mobile/tablet view
![image](https://github.com/Automattic/jetpack/assets/65001528/4399c42d-fbd6-41e3-8ff5-1758dbb2ed51)
![image](https://github.com/Automattic/jetpack/assets/65001528/ebd430a7-c7f5-4c8a-80e8-0569c0c543ab)
7. Make sure video plays correctly, and make sure the CTA works correctly
